src/Raven.Server/Storage/Schema/UpdateStep.cs

```diff
@@ -19,8 +21,15 @@ public UpdateStep(SchemaUpgradeTransactions transactions)
         public DocumentsStorage DocumentsStorage;
         public ClusterStateMachine ClusterStateMachine;
 
-        public void Commit()
+        public void Commit(DocumentsOperationContext txContext)
         {
+            if (txContext != null)
+            {
+                if (txContext.GetTransactionMarker() == DocumentsOperationContext.DefaultTransactionMarker)
```

Comment:
`2` is a valid number to have during normal operation

Reply:
yes. I've forcepushed the fix

Reply:
and now i'm checking that it's not the same as in previous tx